src/style.css

```
@@ -0,0 +1,24 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.box {
+  display: flex;
+}
+
+.stars {
+  background-image: url(images/star.svg);
+  color: transparent;
+  height: 12px;
+  width: 14px;
+  margin: 0.2em;
```

better define a one-direction margin;
and i guess the width and height could be 16px each
```suggestion
  height: 16px;
  width: 16px;
  margin-right: 4px;
```

and remove the margin-right with the `last-child` selector - to not have that last margin stick-out

---

understood. thanks